ci/infrastructure/scripts/infrastructure.sh

```
@@ -254,6 +254,22 @@ defaultSettings() {
     VS_CONTAINER_NAME_BASE=$(dirname "$JOB_NAME" | sed -e "s/\//_/g")
     VS_CONTAINER_NAME_SHORT=$(echo "$BRANCH_NAME" | sed -e "s/\//_/g")
     VS_CONTAINER_NAME_SHORTEST=$(basename "$BRANCH_NAME")
+    echo VS_CONTAINER_NAME_SHORTEST
+    echo $VS_CONTAINER_NAME_SHORTEST
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)